src/load-config.js

```diff
@@ -2,9 +2,8 @@ import fs from 'node:fs';
 import path from 'node:path';
 import { fileURLToPath } from 'node:url';
 
-export const projectRoot = path.dirname(
-  path.dirname(fileURLToPath(import.meta.url))
-);
+const projectRoot = path.dirname(path.dirname(fileURLToPath(import.meta.url)));
+export const sqlDir = path.join(projectRoot, 'src', 'sql');
```

Comment:
At first glance, this feels like a change in path.
For example, [up here](https://github.com/actualbudget/actual-server/pull/121/files#diff-95e90f57d7a98dcb3cee08f7082b6dfcbd59d1205bc80d5766cc5a8e159e38c5L21) isn't the old version `<projectRoot>/sql/account.sql` and the new `<projectRoot>/src/sql/account.sql`?

If I have that right, is that intentional?

Reply:
Yes, I moved the .sql files into `src/` so that the containers only have to copy `app.js` and the `src` folder into the container (plus `package.json` and `yarn.lock` for dependencies)

Reply:
Ah, I see how that's related now. That answers my approval comment meta-question. Thanks.